.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+ - "3.6.1"
+# Install dependencies
+install:
+  - pip install -r requirements.txt
+  - pip install coveralls
+# Run tests
+script:
+  nosetests --with-coverage --cover-package=tests && coverage report
```

Comment:
`--cover-package=tests` will only show you coverage results for the tests folder. Try `--cover-package=app` to see coverage for code inside of the app folder